File: src/media/Track.java
package media;

import java.util.List;

/**
 * A single media file.
 */
public class Track {
    private final Media media;
    private final TrackMetadata metadata;
    private final TrackOptions options;

    Track(Media media, TrackMetadata metadata, TrackOptions options) {
        this.media = media;
        this.metadata = metadata;
        this.options = options;
    }
}

File: muzak/media/Track.cs
namespace muzak.media
{
    /** A single media file? */
    class Track
    {
        private readonly Media media;
        private readonly TrackMetadata metadata;
        private readonly TrackOptions options;

        Track(Media media, TrackMetadata metadata, TrackOptions options)
        {
            this.media = media;
            this.metadata = metadata;
            this.options = options;
        }
    }
}
